ledger-core/ledger/server/buildersvc/service.go

```diff
@@ -34,18 +34,22 @@ type AppendFuture interface {
 }
 
 type Service interface {
-	CreatePlash(node.ShortNodeID, pulse.Range, jet.Tree, census.OnlinePopulation) (PlashAssistant, []jet.ExactID)
+	CreatePlash(pulse.Range, jet.Tree, census.OnlinePopulation) (PlashAssistant, []jet.ExactID)
```

Comment:
There is no word 'Plash' in Cambridge dictionary =(

Reply:
https://www.merriam-webster.com/dictionary/plash

Reply:
https://dictionary.cambridge.org/dictionary/english/plush

you have to look for full english dictionary, not for essential one

Reply:
free version of Cambridge dictionary seems to be limited ...